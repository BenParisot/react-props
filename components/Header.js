import React from 'react';

export default function Header() {
  return (
        <>
            <img src="https://image.spreadshirtmedia.com/image-server/v1/mp/designs/1010111384,width=178,height=178,version=1487039039/smile-icon-facebook.png" />
            <h1>Check out this dog yo</h1>
        </>
  );
}
