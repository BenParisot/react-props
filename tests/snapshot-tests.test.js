import React from 'react';
import { shallow } from 'enzyme';
import Dog from '../components/Dog';
import Header from '../components/Header';

describe('Dog component', () => {
  it('renders a Dog', () => {
    const wrapper = shallow(<Dog name="Joe" age={5} weight="90 lbs" />);
    expect(wrapper).toMatchSnapshot();
  });
});

describe('Header component', () => {
  it('renders a Header', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
