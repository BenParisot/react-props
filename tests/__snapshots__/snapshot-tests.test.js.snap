// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dog component renders a Dog 1`] = `
<dl>
  <dt>
    Name
  </dt>
  <dd>
    Joe
  </dd>
  <dt>
    Age
  </dt>
  <dd>
    5
  </dd>
  <dt>
    Weight
  </dt>
  <dd>
    90 lbs
  </dd>
</dl>
`;

exports[`Header component renders a Header 1`] = `
<Fragment>
  <img
    src="https://image.spreadshirtmedia.com/image-server/v1/mp/designs/1010111384,width=178,height=178,version=1487039039/smile-icon-facebook.png"
  />
  <h1>
    Check out this dog yo
  </h1>
</Fragment>
`;
